Simplified/Accounts/Library/AccountsManager.swift

```diff
@@ -23,6 +23,8 @@ let currentAccountIdentifierKey  = "NYPLCurrentAccountIdentifier"
   ]
 
   let NYPLAccountUUID = AccountsManager.NYPLAccountUUIDs[0]
+  
+  let SimplyEAccountUUID = AccountsManager.NYPLAccountUUIDs[2]
```

Comment:
ah I see,... this sounds good for now, ideally NYPLAccountUUID and SimplyEAccountUUID should be static, but don't worry about it for now. AccountsManager needs a refactor as a whole, it's the origin of many evils in the app.

Reply:
Agree with @ettore on this one.